pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+    """Manages the parameters used for computing the mean/variance."""
+    eps: float
+    delta: float
+    low: float
+    high: float
+    max_partitions_contributed: int
+    max_contributions_per_partition: int
+    noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+    def l0_sensitivity(self):
+        return self.max_partitions_contributed
+
+    def linf_sensitivity(self, metric):
+        return self.max_contributions_per_partition * {
+            pipeline_dp.Metrics.COUNT: 1,
+            pipeline_dp.Metrics.SUM: np.max(self.low, self.high)
+        }.get(metric)
+
+
+def _l1_sensitivity(l0_sensitivity: float, linf_sensitivity: float):
+    return l0_sensitivity * linf_sensitivity
+
+
+def _l2_sensitivity(l0_sensitivity: float, linf_sensitivity: float):
+    return np.sqrt(l0_sensitivity) * linf_sensitivity
+
+
+def _compute_sigma(eps: float, delta: float, l2_sensitivity: float):
+    # Theorem 3.22: https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf
+    return np.sqrt(2 * np.log(1.25 / delta)) * l2_sensitivity / eps
+
+
+def _apply_laplace_mechanism(value: float, eps: float, l1_sensitivity: float):
+    # TODO: use the secure noise instead of np.random
+    return value + np.random.laplace(0, l1_sensitivity / eps)
+
+
+def _apply_gaussian_mechanism(value: float, eps: float, delta: float, l2_sensitivity: float):
+    sigma = _compute_sigma(eps, delta, l2_sensitivity)
+
+    # TODO: use the secure noise instead of np.random
+    return value + np.random.normal(0, np.power(sigma, 2) * (eps, delta, l2_sensitivity))
```

Comment:
1.It looks like we don't need (eps, delta, l2_sensitivity) in this line
2.np.power(sigma, 2) -> sigma (according to [doc](https://numpy.org/doc/stable/reference/random/generated/numpy.random.normal.html) np.random.normal parameter is std, i.e. sigma